### ISSUE TITLE
Get the website on the HSTS preload list

### ISSUE THREAD

**Issue by ExE-Boss** [NOVAAPI/novaapi.github.io#16]
The [HTTP Strict Transport Security (HSTS) preload list](https://hstspreload.org) is a list of websites that is used by modern major browsers to know which websites are to be loaded only over HTTPS, in order to get the [NOVA website](https://novaapi.net/) on there, first issues #14 and #15 have to be resolved.

Another issue is that by putting the NOVA website on the HSTS preload list, we will be enforcing HTTPS on [Artifactory](https://artifactory.novaapi.net), [Jenkins](https://ci.novaapi.net) and [Maven](https://maven.novaapi.net).

### Blocked by:
- #14
- #15
